src/components/donationCard/DonationCard.js

```diff
@@ -0,0 +1,160 @@
+          <div className="donation-card__tags">
+            {props.tags.map((tag) => {
+              return (
+                <p
+                  key={tag.name}
+                  className="donation-card__tags donation-card__tags--css"
+                >
+                  {tag.name}
+                </p>
+              );
+            })}
+          </div>
+          <p className="donation-card__text donation-card__text--black">
+            {props.description}
+          </p>
+          <h4>How much do you want to donate?</h4>
+          <div className="donation-card__amount">
+            <aside className="donation-card__amount-display donation-card__amount-display--white">
+              €25
+            </aside>
+            <aside className="donation-card__amount-inputs">
+              {[1, 5, 10, 25, 50, 100, 250].map((amount) => {
+                return (
+                  <button key={amount} className="donation-card__amount-btn">
+                    € {amount}
+                  </button>
+                );
+              })}
+              <label className="form__label--input" htmlFor="amount">
+                <input
+                  className="form__input"
+                  type="number"
+                  placeholder="Enter an amount"
+                  {...register("amount", { required: true })}
+                />
+              </label>
+            </aside>
+          </div>
+          <h5>Leave a comment with your donation (optional)</h5>
+          <div className="donation-card__comment">
+            <label className="form__label" htmlFor="comment">
+              <textarea
+                className="form__input"
+                placeholder="Leave your comment here"
+                {...register("comment", { maxLength: 255 })}
+              />
+            </label>
+            <p className="donation-card__charleft">0/255 characters</p>
+          </div>
+          <div className="donation-card__btn">
+            <Button
+              type="submit"
+              icon={
+                <FaHandHoldingHeart
+                  style={{
+                    color: "white",
+                    fontSize: "0,5em",
+                    marginRight: "0.5rem",
+                  }}
+                />
+              }
+              text={`support ${props.by} with €€€€`}
+              size="medium"
+              variant="purple"
+            />
+          </div>
+          <div className="donation-card__payment">
+            <h4 className="donation-card__ideal">
+              Payment methods:{" "}
+              <FaIdeal style={{ fontSize: "2em", marginLeft: "0.5rem" }} />
+            </h4>
+          </div>
+        </form>
+        <div className="donation-card__footer">
+          <div>
+            <GiPiggyBank
+              style={{
+                color: "#751cc1",
+                fontSize: "2em",
+                marginBottom: "0.5rem",
+              }}
+            />
+            <p>€ amount</p>
```

Comment:
This amount is not dynamic yet (and the rest of the figures in the footer aren't either

Made a card in trello for this
<img width="204" alt="Screenshot 2021-07-19 at 11 39 59" src="https://user-images.githubusercontent.com/20372832/126140216-c698381d-0b6f-4524-8ce8-1f693968e067.png">